src/lang/std/artifactGraph.test.ts

```diff
@@ -458,6 +458,9 @@ async function GraphTheGraph(
   await page.waitForSelector('#plotly-graph')
   const element = await page.$('#plotly-graph')
 
+  // wait an extra bit for things to settle
+  await new Promise((resolve) => setTimeout(resolve, 500))
```

Comment:
Unrelated to the change, but I've seen the artifact graph snap shots update a few times because of very minor artifacts in the images changing, and just trying to stabilise them by adding a bit of a wait in, don't know if it will work, but kinda have to test in CI since they're stable when running this locally.